### DISCUSSION TITLE
Show all points on the map

### DISCUSSION THREAD

**Discussion by Tazmanian79** [Freika/dawarich#67] [Ideas]
In Google Timeline I could see all the points on the maps where I have been.  But not all the points does show up in the DaWarich Map.
I think a point is only shown when you have been there longer than "MIN_MINUTES_SPENT_IN_CITY".

But I want to see all points, and even draw lines between each map so you can see the 'route' I have taken.
Is that possible?

I can set MIN_MINUTES_SPENT_IN_CITY to 0 but that's not correct.  I want to show up the city when I have really been there, not drove through that city.

**Comment by Freika** [comment]
If you select starting date and ending date to include all period of your tracking history and enable "Points" on the map, you'll be able to see all points. It will probably hang up your browser tab though.

<img width="398" alt="изображение" src="https://github.com/Freika/dawarich/assets/3738638/531a4227-2b25-40a9-a5f8-205a952647ac">

Routes, on the other side, are being split based on time or distance between points, you can read more about it on [0.5.0 release page](https://github.com/Freika/dawarich/releases/tag/0.5.0). `MIN_MINUTES_SPENT_IN_CITY` env variable has nothing to do with routes rendering or not, it's only responsible for showing or not showing cities in right sidebar on the map page and at the moment these cities are being calculated in a suboptimal way, which I have intention to fix at some point.

Previously, routes were not being splitted and whole map could have been zigzagged with lines, connecting consecutive points. I decided not to render all the lines and render only the ones that connect closest points both in time and distance so they show a real route.

I want to allow user to change criteria for splitting routes in terms of minutes and meters between consecutive points, but at the moment it can be done only with url params, which are also described on 0.5.0 release page.
